pyatlan/model/search.py

```diff
@@ -1803,9 +1804,8 @@ class DSL(AtlanObject):
     track_total_hits: bool = Field(True, alias="track_total_hits")
     post_filter: Optional[Query] = Field(alias="post_filter")
     query: Optional[Query]
-    sort: list[SortItem] = Field(
-        alias="sort", default=[SortItem(TermAttributes.GUID.value)]
-    )
+    sort: list[SortItem] = Field(alias="sort", default_factory=list)
+    _req_class_name: ClassVar[str]
```

Comment:
Why is this a class variable? This means there is only one value for the DSL class. This will end up having the last value set. I think this should be a normal required attribute